universe/vault/types.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Namespace Labs Inc; All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
+
+package foundation.universe.vault;
+
+option go_package = "namespacelabs.dev/foundation/universe/vault";
+
+message Certificate {
+    string               base_domain = 1;
+    string               region      = 2;
+    CertificateAuthority ca          = 3;
+}
+
+message CertificateAuthority {
+    string vault_addr      = 1;
```

Comment:
No strong opinion, but prefer not have have lots of nested objects.
What would be a benefit of having `vault` submessage.

Reply:
With fresh mind, let's skip the sub message, since `vault` is already in the package.

Let's spell out `address` instead of `addr` for consistency with other protos.

Reply:
changed